go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	k8s.io/kube-aggregator v0.26.2
 	k8s.io/utils v0.0.0-20230202215443-34013725500c
 	kubevirt.io/api v0.59.0
-	kubevirt.io/containerized-data-importer-api v1.55.2
+	kubevirt.io/containerized-data-importer-api v1.57.0-alpha1
```

Comment:
Is possible to use stable release instead of alpha?

Reply:
we should have one pretty soon, and then we can switch

Reply:
@akalenyu Will you change it in another PR? If yes, is there an issue or ticket to track it?

Reply:
usually prefer to pull in releases instead of stray commits which we are doing, but I can open an issue here to track this
@lyarwood wdyt?